extensions/users-permissions/documentation/1.0.0/users-permissions-User.json

```diff
@@ -1338,6 +1336,15 @@
                 "newMeetingTimes": {
                   "type": "component"
                 },
+                "opportunities": {
+                  "type": "array",
+                  "items": {
+                    "type": "string"
+                  }
+                },
+                "isPlatform": {
```

Comment:
What is this field for?

Reply:
So during one of the leads meetings we had a discussion about "platform vs independent" products

Platform products are tied to Dev Launchers as a brand so they have to help Dev Launchers reach its goals. In exchange they'll have special place in Dev Launchers official website. An example would be "DevRecruiters" which is an app that helps people who are willing to volunteer to find opportunities within Dev Launchers products.

Independent products are not tied to Dev Launchers and may not follow the branding of it but they're products accelerated by Dev Launchers through giving them an infrastructure of building a software, easier way to find volunteers, cost free deployment and integration.
An example would be DevBots (the only independent product so far).